tests/unit/amo/api/test_abuse.js

```diff
@@ -92,19 +92,110 @@ describe(__filename, () => {
           auth: true,
           endpoint: 'abuse/report/user',
           method: 'POST',
-          body: { message, user: '5001' },
+          body: {
+            user: '5001',
```

Comment:
nit: the norm for frontend tests is to have a reused value defined as a constant like `const reportedUserId = 5001` rather than hard coding it everywhere.